eva-accession-ws/src/test/java/uk/ac/ebi/eva/accession/ws/ClusteredVariantsRestControllerTest.java

```diff
@@ -154,12 +169,23 @@
 
     private DbsnpClusteredVariantEntity clusteredVariantEntity3;
 
+    private DbsnpClusteredVariantEntity clusteredHumanVariantEntity1;
+
+    private DbsnpClusteredVariantEntity clusteredHumanVariantEntity2;
+
+    private DbsnpClusteredVariantEntity clusteredHumanVariantEntity3;
+
+    private DbsnpClusteredVariantOperationEntity clusteredVariantOperationEntity1;
```

Comment:
Suggest making this local to the method which uses this since it is only being used for the insertion operation.